src/elements/content-uploader/__tests__/ItemResume-test.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import noop from 'lodash/noop';
+import { shallow } from 'enzyme';
+import ItemResume from '../ItemResume';
+import { STATUS_PENDING, STATUS_IN_PROGRESS, STATUS_COMPLETE, STATUS_ERROR } from '../../../constants';
+
+describe('elements/content-uploader/ItemResume', () => {
+    const getWrapper = props => shallow(<ItemResume onClick={noop} status={STATUS_PENDING} {...props} />);
+
+    test('should render correctly with STATUS_COMPLETE', () => {
```

Comment:
Consider using `jest.each` for tests where there's only one or two inputs and outputs. We should also tend toward writing tests with specific expectations rather than snapshots, as changes to the latter tend to be ignored.